tools/dev/msftidy.rb

```diff
@@ -65,6 +65,7 @@ class Msftidy
 
   def initialize(source_file)
     @full_filepath = source_file
+    @module_type = File.dirname(File.expand_path(@full_filepath))[/\/modules\/([^\/]+)/, 1]
```

Comment:
I'm not thrilled with this, but I'm not sure of a way to determine the _intended_ module type other than by looking at the directory it lives in.  This assumes any module passed to `msftidy` is under a `modules` directory.